src/main/java/org/chiclepad/backend/Dao/ChiclePadUserDao.java

```diff
@@ -0,0 +1,157 @@
+package org.chiclepad.backend.Dao;
+
+import org.chiclepad.backend.LocaleUtils;
+import org.chiclepad.backend.entity.ChiclePadUser;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Locale;
+
+public class ChiclePadUserDao {
+
+   // Dara source
+   private JdbcTemplate jdbcTemplate;
+
+   protected ChiclePadUserDao(JdbcTemplate jdbcTemplate) {
+      this.jdbcTemplate = jdbcTemplate;
+   }
+
+   //CREATE
+   public ChiclePadUser create(String email, String password, String salt) throws DuplicateKeyException {
+
+      String sqlInsert = "INSERT INTO chiclepad_user(id,email, password, salt)"
+            + " VALUES(DEFAULT ,?,?,?) RETURNING id ;";
+
+      Object id = null;
+      try {
```

Comment:
`try { <code> } catch(.....) {.....}` -> `<code>`

The method should throw in case the key already exists, not tell the `stdout` that is the case.